cmd/registry/main.go

```diff
@@ -25,6 +27,15 @@ func main() {
 	zkConfig := kafkazk.Config{}
 	adminConfig := admin.Config{Type: "kafka"}
 
+	securityProtocols := make([]string, 0, len(kafkaadmin.SecurityProtocolSet))
+	for k := range kafkaadmin.SecurityProtocolSet {
+		securityProtocols = append(securityProtocols, k)
+	}
+	saslMechanims := make([]string, 0, len(kafkaadmin.SASLMechanismSet))
```

Comment:
I wouldn't bother changing but fwiw you don't need to set slice sizes at instantiation if you're going to use `append`; it can safely grow the underlying array: https://play.golang.org/p/AMtFBNK17aS

Reply:
yes, but it will trigger a realloc underneath every-time we hit capacity in the slice which affects performance. In this case prob is negligible as the array is very small.